app/backplane/power_module/src/c_power_module.cpp

```diff
@@ -1,5 +1,7 @@
 #include "c_power_module.h"
 
+#include <n_ac_types.h>
```

Comment:
kinda a nitpick but how do we feel about spelling out the entire `n_autocoded_types`. `ac` can mean a lot of things and isnt very informative if you're just getting familiar with the project. Especially bc the file is auto generated and not findable just by looking at the file structure since it goes to the build dir.

Reply:
Done